README.md

```diff
@@ -129,102 +126,134 @@ service:
+
+```
+fern login
 ```
 
-To generate the TypeScript SDK, run:
+You'll be prompted to log in to your GitHub account using your preferred browser.
 
-```bash
+After you have logged in, simply run the generate command to generate your SDKs:
+
+```
 fern generate
 ```
 
-And voila! You just built and published a TypeScript SDK.
+After your code is generated, the resulting artifacts are copied to the output directory specified in `generators.yml` , which we will touch on in the next section.
 
-**Next step:** define _your_ API in Fern. Check out our [docs](https://www.buildwithfern.com/docs/definition) to learn more.
+It is possible to generate the code on your own machine by using the `--local` flag. This does require you to run Docker on your local machine, and Fern will schedule containers to do the code generation automatically. This is further specified in the CLI reference below.
 
-# CLI reference
+## Adding Generators
 
-### `fern generate [--group <group>] [--version <version>]`
+You can add generators using the `fern add`. You can have multiple generators for a single project, e.g. a single `fern generate` call could generate SDKs in Python, TypeScript and Java all in one go.
 
-`fern generate` runs the compiler. It will validate your API and run the
-generators specified in `generators.yml`.
+For instance, if you'd like to add a TypeScript generator to your project, simply run:
 
-If you don't have a `default-group` specified in `generators.yml`, then you must
-specify a group using the `--group` option.
+```
+fern add fern-typescript-sdk
+```
 
-You can specify a version using the `--version` option. This version string is
-used when publishing SDKs to registries (e.g. npm).
+Which will add the Node variant of the TypeScript SDK to the list of generators - the other variant being a TypeScript browser library, to be used by frontends directly.
 
-#### Running locally
+Your `generators.yml` file should contain the new generator:
 
-By default, the generation runs in the cloud. If you want to run it on your local machine, you can use the `--local` option. This will run each generator in a Docker container.
+```
+default-group: local
+groups:
+  local:
+    generators:
+      - name: fernapi/fern-typescript-node-sdk
+        version: 0.7.1
+        output:
+          location: local-file-system
+          path: ../../generated/typescript
+```
 
-By default, Fern will delete the container after running. To keep the container around (e.g. to look at the generator's logs), use the `--keepDocker` option.
+Note the `local-file-system` output may not be present by default. If it's not, simply add it and specify a local path to where the generated client will be saved.
 
----
+## Publishing Clients
 
-### `fern check`
+Fern supports automatically publishing clients to an NPM registry. By default, Node packages are published to `npm.buildwithfern.com` and are named `@organization-fern/package`
```

Comment:
can we embed an example config of `generators.yml` here? 

https://github.com/fern-api/fern-definition-registry/blob/4aabec28789ba7de517159053f2568b95803b46c/fern/api/generators.yml#L21-L26

Reply:
Done. Should I mention that different publishing targets are part of the paid version of Fern?